config/src/config/mempool_config.rs

```diff
@@ -60,9 +60,9 @@ pub struct MempoolConfig {
     pub broadcast_buckets: Vec<u64>,
     pub eager_expire_threshold_ms: Option<u64>,
     pub eager_expire_time_ms: u64,
-    /// Uses the MempoolSyncMessageV2 instead of MempoolSyncMessage when sending mempool transactions
+    /// Uses the BroadcastTransactionsRequestWithReadyTime instead of MempoolSyncMessage when sending mempool transactions
```

Comment:
nit: remove `MempoolSyncMessage` -> `BroadcastTransactionsRequestWithReadyTime instead of BroadcastTransactionsRequest`